.github/workflows/ci.yml

```
@@ -0,0 +1,43 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - actions/checkout@v2.3.4
+      - actions/setup-node@v2.1.2
+      - yarn install --frozen-lockfile
+      - yarn run lint
```

Ref: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idsteps

I guess the `steps` are required to be a sequence of mapping:

```suggestion
      - uses: actions/checkout@v2.3.4
      - uses: actions/setup-node@v2.1.2
      - run: yarn install --frozen-lockfile
      - run: yarn run lint
```